### PR TITLE
add other icon

### LINKED ISSUE #1218: Other icon under resource class
I suggest a question mark (although seems more appropriate for unknown) or a simple but thick dash for the other icon under resource class.

### PR DESCRIPTION
**Pull request by dnoneill**
closes #1218

### CODE REVIEW

File: app/components/blacklight/icons/other_component.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Blacklight
+  module Icons
+    class OtherComponent < Blacklight::Icons::IconComponent
```
Comment (thatbudakguy) [inline app/components/blacklight/icons/other_component.rb:5]:
```suggestion
    class OtherResourceClassComponent < Blacklight::Icons::IconComponent
```
less confusing?

Comment (dnoneill) [reply app/components/blacklight/icons/other_component.rb:5]: We will have to add a mapping into the config if we do this.

Comment (thatbudakguy) [reply app/components/blacklight/icons/other_component.rb:5]: I guess it's already in the `Icons` namespace, so not as bad as it could be...

**Review by thatbudakguy — APPROVED** [review]

File: app/components/blacklight/icons/other_component.rb (hunk 1 above)
Comment (dnoneill) [reply app/components/blacklight/icons/other_component.rb:5]: We can do it, we would have to add the line 
other-resource: other here:
https://github.com/sul-dlss/earthworks/blob/bl8/config/settings.yml#L465-L466

**Pull request merged**